### PR TITLE
Component Setup

### PR DESCRIPTION
**Pull request by Obimbo07**
This pull request introduces the following:  

- [ ] Add 2 routes in the application, each should render a page component:
             /
             /categories

- [ ] Create components inside of the /components directory:

> Contains individual book state, as well as a button to delete the book
> Contains a list to renders individual books
> Contains a form to create new books
> Contains a navigation to link to the 2 routes you created

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,165 @@
...
+I would like to thank Microverse for providing the UI
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
 
-### `npm run build` fails to minify
+This project is [MIT](./LICENSE) licensed.
```
Comment (Bantealem) [inline README.md:163]: Great job so far. The link for the `MIT` License is currently broken. Please fix it.

**Review by Bantealem — CHANGES_REQUESTED** [review]
# Status: Change Required
Hi @Obimbo07, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Linters are passed
- [x] Correct Git flow
- [x] Descriptive PR summary

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Obimbo07) [reply README.md:163]: Done, thank you for the reminder.

**Review by Bantealem — APPROVED** [review]
# Status: Approved
Hi @Obimbo07, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Screenshot from 2023-09-27 20-56-29


## Highlights
- [x] Correct Git flow
- [x] Professional Readme
- [x] Descriptive PR summary
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**